components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceDataHolder.java

```diff
@@ -607,4 +609,27 @@ public void setAdaptiveAuthenticationAvailable(boolean adaptiveAuthenticationAva
 
         isAdaptiveAuthenticationAvailable = adaptiveAuthenticationAvailable;
     }
+
+    /**
+     * Get is organization management enabled.
+     *
+     * @return True if organization management is enabled.
+     */
+    public boolean isOrganizationManagementEnable() {
```

Comment:
```suggestion
    public boolean isOrganizationManagementEnabled() {
```

Reply:
fixed in [077830a](https://github.com/wso2/carbon-identity-framework/pull/4165/commits/077830a9e7ab93ed3531984fe2f9c7ce154b8ab2)